### PR TITLE
Event update

### PR DESCRIPTION
**Pull request by reginafcompton**
@fgregg - this pull request address issue #154. It includes a couple tests, using pytest. (N.B. I had some simple tests for manual agenda upload, written with django unittest. So, I had to make some large deletes to get pytests up-and-running.)

### CODE REVIEW

File: lametro/templatetags/lametro_extras.py [hunk 1]
```diff
@@ -144,4 +144,16 @@ def parse_agenda_item(text):
...
\ No newline at end of file
+        return ''
+
+
+@register.filter
+def updates_made(event_id):
+    event = Event.objects.get(ocd_id=event_id)
+
+    try:
+        document = EventDocument.objects.get(event_id=event_id)
```
Comment (fgregg) [inline lametro/templatetags/lametro_extras.py]: There is only one document for every event?

File: lametro/templatetags/lametro_extras.py [hunk 2]
```diff
@@ -144,4 +144,16 @@ def parse_agenda_item(text):
...
+    except EventDocument.DoesNotExist:
+        return False
+    else:
+        return True if document.updated_at < event.updated_at else False
```
Comment (fgregg) [inline lametro/templatetags/lametro_extras.py]: It's clearer to do this like

```python
return document.updated_at < event.updated_at
```

and even clearer to do

```python
updated = document.updated_at < event.updated_at
return updated 
```

File: lametro/templatetags/lametro_extras.py [hunk 3]
```diff
@@ -144,4 +144,16 @@ def parse_agenda_item(text):
         label, number = text.split(',')
         return number
     else: 
-        return ''
\ No newline at end of file
+        return ''
+
+
+@register.filter
+def updates_made(event_id):
```
Comment (fgregg) [inline lametro/templatetags/lametro_extras.py]: Please add a docstring to explain why we want to know when an event was updated after a document associated with that event?

**Comment by reginafcompton** [comment]
@fgregg - changes made to the template tag. Let me know how it looks!

**Pull request merged**